Modules/Synchronization/src/Synchronization.Domain/Entities/DatawalletModification.cs

```diff
@@ -6,10 +6,13 @@ namespace Backbone.Modules.Synchronization.Domain.Entities;
 public class DatawalletModification
 {
 #pragma warning disable CS8618
-    private DatawalletModification() { }
+    private DatawalletModification(string blobReference)
```

Comment:
Why the new parameter in the private construtor when it's only used by EF Core to generate the migrations?

Reply:
Oh, it seems ReSharper added it to all constructors and not just the "real" one